Mathlib/Algebra/BigOperators/Pi.lean

```diff
@@ -146,3 +144,57 @@ theorem snd_prod : (∏ c in s, f c).2 = ∏ c in s, (f c).2 :=
 #align prod.snd_sum Prod.snd_sum
 
 end Prod
+
+open Finset
+
+namespace Fintype
+variable {ι κ : Type*} {α : ι → Type*} [DecidableEq ι] [Fintype ι] [DecidableEq κ]
+  [∀ i, DecidableEq (α i)]
+
+lemma card_filter_piFinset_eq_of_mem [∀ a, DecidableEq (α a)] (s : ∀ i, Finset (α i)) (i : ι)
```

Comment:
This one might be worthy of a docstring

Reply:
Actually I found a nice lemma to abstract from the proof, but it's not nice not to need this lemma.